frontend/src/App.js

```diff
@@ -5,7 +5,23 @@ import Navigation from './components/Navigation';
 import Trips from './screens/Trips';
 import AddTrip from './components/AddTrip';
 import Notification from './components/Notification';
-import theme from './themes/base';
+import Login from './components/Login';
+
+/**
+ * Access like this:
+ * const MyFancyButton = styled.button`
+ * background: ${props => props.theme.colors.primary}
+ * `
+ */
+const theme = {
+  colors: {
+    primary: '#8064f7',
+    secondary: '#f5ba9c',
+    fill: '#1a1a1a',
+    alternateFill: '#f0ebff',
+    inactive: '#707386',
+  },
+};
```

Comment:
Den här delen är inlagd i /themes/base och kan tas bort.

Reply:
Har ni ändra nåt på master, för att igår kollade upp master och den delen var redan inlagd på app.js master?

Reply:
Jag tog bort den delen bara, sen gjorde commit men gick inte genom testet fast den gick igår, och jag har bara tagit bort från line 10 till 24 som du sa

Reply:
Den säger them is not defined på testet, och du säger att den är inlagd i themes/base, och jag kan ta bort den, vet inte vad jag behöver göra ! :(

Reply:
Du måste lägga till `import theme from './themes/base';`, kolla alltid i "Files changed" längst upp för att se vad du har ändrat på förhållande till det som ligger på master. I "Files changed" står även vad för problem med lint etc du måste lösa.

Reply:
Jag tror att den är fixat nu!